sdk/src/test/scala/com/horizen/SidechainNodeViewHolderTest.scala

```diff
@@ -42,6 +47,7 @@ class SidechainNodeViewHolderTest extends JUnitSuite
 
   val genesisBlock: SidechainBlock = SidechainBlockFixture.generateSidechainBlock(sidechainTransactionsCompanion)
   val params: NetworkParams = RegTestParams(initialCumulativeCommTreeHash = FieldElementFixture.generateFieldElement())
+  implicit lazy val timeout: Timeout = Timeout(100000 milliseconds)
```

Comment:
do we need it to be 100s? In case of any bug here, we may wait 100s to fail the test.
Let's have 10s, we still will be safe even when running this test on the weak PC.